mods/ra/weapons/other.yaml

```diff
@@ -170,6 +170,36 @@ Mandible:
 			None: 90
 		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
 
+MandibleHeavy:
+	Inherits: Claw
+	ReloadDelay: 15
```

Comment:
The ant attack sound is missing. Not sure how annoying it might get due to the low reload delay, though.

Sound name is `antbite`.

Same goes for `mandible` and `AntiFireball`.

Reply:
Ok, I tested and it's definitely annoying. How should I handle this?

Reply:
What did the original do? Was it that annoying there as well?

Reply:
Not sure, I think in the original the sound was tied to the attack animation, which was slower than the hypothetical ROF.

We could try giving the mandible weapons Burst: 2, with BurstDelays set to 14 (needs to be lower than ReloadDelay) so players won't notice, and then use `StartBurstReport` instead of Report, so it only plays every 2nd attack.

Reply:
@reaperrr Ok, I tried it out and it worked out nicely